src/main/java/org/opentripplanner/transit/raptor/rangeraptor/multicriteria/McTransitWorker.java

```diff
@@ -100,6 +100,7 @@ public void board(
                 final int boardWaitTimeForCostCalculation = boardTime - prevArrival.arrivalTime();
         final int relativeBoardCost = calculateOnTripRelativeCost(
             prevArrival,
+            trip.transitReluctanceFactorIndex(),
```

Comment:
The indentation for line 100 is a bit off, could that also be corrected?

Reply:
Fixed.